File: for.p.js
const isPrime = (n) => {
	for (let i = 3; i < n / 2; i += 2) {
		if (n % i === 0) {
			return false;
		}
	}

	return true;
};

const max = +process.argv[process.argv.length - 1];

const start = Date.now();
let prime = 0;

for (let i = 1; i < max; i += 2) {
	if (isPrime(i)) {
		prime++;
	}
}

console.log(Date.now() - start);

File: for.p.c
#include <time.h>
#include <stdlib.h>
#include <stdio.h>
#include <stdbool.h>

bool is_prime(int n)
{
	for (int i = 3; i < n / 2; i += 2)
	{
		if (n % i)
		{
			return false;
		}
	}

	return true;
}

int main(int argc, char *argv[])
{
	int max = atoi(argv[argc - 1]);

	clock_t start = clock();

	volatile int prime = 0;

	for (int i = 1; i < max; i += 2)
	{
		if (is_prime(i))
		{
			prime++;
		}

		i += 2;
	}

	printf("%f\n", (double)(clock() - start) / CLOCKS_PER_SEC * 1000);

	return 0;
}
